content/articles/2019-09-pkgdown-1-4-0.Rmarkdown

```diff
@@ -0,0 +1,60 @@
+---
+title: pkgdown 1.4.0
+author: Hadley Wickham
+date: '2019-09-03'
+slug: pkgdown-1-4-0
+photo:
+  url: https://unsplash.com/photos/ytMmedc6y44
+  author: noodle kimm
+categories:
+  - package
+tags:
+  - pkgdown
+---
+
+We're happy to announce that [pkgdown](https://pkgdown.r-lib.org/) 1.4.0 is now
+available on CRAN. pkgdown is designed to make it quick and easy to build a
+website for your package. 
+
+## Development and deployment modes
+
+The main change in this version is that `build_site()`, `build_reference()` and  `build_home()` gain a `devel` parameter that controls whether you're in deployment or development mode. 
```

Comment:
Is there magical linking here? If not, I usually link to the function reference on first mention.

Reply:
I'd be willing to waive that preference in this case, since it's pretty function-laden prose.